Makefile

```diff
@@ -2,13 +2,21 @@
 
 TAG := $(shell git log --format=%h -1)
 IMAGE=taccwma/imageinf
+IMAGE_CLIENT  = taccwma/imageinf-ui
```

Comment:
Either change this to taccaci or change the docker-compose.yml to taccwma/imageinf:local

Reply:
It looks like taccwma/imageinf was changed most recently

Reply:
Thanks

> change the docker-compose.yml to taccwma/imageinf:local

Updated